src/error.rs

```diff
@@ -124,6 +118,48 @@ pub enum ConversionsError {
     UnsupportedCodec(multicodec::Codec),
 }
 
+/// SSH Encoding Errors that cannot be handled by thiserror since they may not use the std feature
+/// in the case of wasm32 target.
+#[derive(Clone, Debug)]
```

Comment:
Ah yes, good point! Fixed.

Reply:
Sure, `thiserror` works great when `ssh-key` has `std` feature enabled.

But when disabled, as in this PR, `StdError` and `AsDynError` are not implement on the `ssh_key` or `ssh_encoding` creates, which cause the `wasm32` builds to fail.

Try doing this with those `thiserror` changes:

```
cargo build --target wasm32-unknown-unknown --features wasm
```

Since `thiserror` cannot be used, they are implemented as you see, with all the `e.into()` conversions to get them from `SshError` into a `ConversionError`. `Display` transparently passes the source up, same as `thiserror`. 

The code is correct in order to build to `wasm32`.